AsyncDisplayKit/Layout/ASStackLayoutSpec.h

```diff
@@ -35,10 +35,25 @@
  */
 @interface ASStackLayoutSpec : ASLayoutSpec
 
-/** Specifies the direction children are stacked in. */
+/** 
+ Specifies the direction children are stacked in. If horizontalAlignment and verticalAlignment were set, 
+ they will be resolved again, causing justifyContent and alignItems to be updated accordingly
+ */
 @property (nonatomic, assign) ASStackLayoutDirection direction;
 /** The amount of space between each child. */
 @property (nonatomic, assign) CGFloat spacing;
+/** 
+ Specifies how children are aligned horizontally. Depends on the stack direction, setting the alignment causes either
+ justifyContent or alignItems to be updated. The alignment will remain valid after future direction changes.
+ Thus, it is preferred to those properties
```

Comment:
Very minor - this sentence wasn't completed.  Let's patch it up in a future diff (it doesn't have to be specific to this, could even be a change to an example app).  Is this format readable by AppleDoc?